modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss

```diff
@@ -0,0 +1,16 @@
+@import 'atlas-variables';
+
+.lfr-objects__model-builder-left-sidebar {
+	display: flex;
+	flex-direction: column;
+	padding: 1rem;
+
+	.navbar-form,
+	.container-fluid {
+		padding: 0 !important;
+	}
+}
+
+.lfr__objects-custom-vertical-bar--left-open {
```

Comment:
@igor-franca we should keep a pattern of using either `lfr-objects__` or `lfr__objects-`

Reply:
Thanks for this review! Changes have been made :)